sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/object.scala

```diff
@@ -519,3 +519,18 @@ case class CoGroup(
     outputObjAttr: Attribute,
     left: LogicalPlan,
     right: LogicalPlan) extends BinaryNode with ObjectProducer
+
+case class FlatMapGroupsInPandas(
```

Comment:
We should add proper comment for `FlatMapGroupsInPandas` and explain its purpose.

Reply:
`FlatMapGroupsInPandas` or `FlatMapGroupsInPython`, which one is proper? We have `FlatMapGroupsInR`.

Reply:
I like `FlatMapGroupsInPandas` a little better because `FlatMapGroupsInPython` doesn't imply it's using a vectorized udf.

Reply:
Yes agreed. I will fix that.

Reply:
Doc added.

Reply:
Hm.. looks the doc for this `case class` is missed?

Reply:
I'd also put this somewhere else that's not object.scala

Reply:
@HyukjinKwon Thanks for catching this. I added docs for `FlatMapGroupsInPandas` (function) `FlatMapGroupsInPandas` (logical node) and `FlatMapGroupsInPandasExec` and cross referenced them.

@rxin I created file `pythonLogicalOperators` under `org.apache.spark.sql.catalyst.plans.logical` and move `FlatMapGroupsInPandas` under that file.